css/home.css

```diff
@@ -0,0 +1,774 @@
+* {
+  margin: 0%;
+  padding: 0%;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Lato", sans-serif;
+  overflow: hidden;
+}
+
+ul {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+/* top menu */
+.top-menu {
+  background: rgb(15, 15, 15);
+}
+
+.top-menu-ul {
+  display: flex;
+  justify-content: flex-end;
+  padding-right: 10%;
+}
+
+.top-menu-ul li {
+  padding: 1%;
+}
+
+.top-menu-ul li a {
+  color: rgb(233, 230, 225);
+}
+
+/* main-menu */
+.main-menu {
+  background: white;
+  height: 15%;
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  padding-left: 8%;
+  padding-bottom: 1.8%;
+}
+
+.main-menu img {
+  width: 80px;
+  height: 60px;
+  margin-top: 1.5%;
+}
+
+.main-menu p {
+  color: rgb(40, 94, 40);
+  font-size: 30px;
+  font-weight: 700;
+  line-height: normal;
+  margin-top: -20%;
+  margin-left: 32%;
+  width: 100%;
+}
+
+.main-menu p span {
+  color: rgb(235, 124, 21);
+}
+
+.main-menu-ul {
+  display: flex;
+  flex-direction: row;
+  width: 40%;
+}
+
+.main-menu-ul li {
+  padding-top: 7%;
+  padding-bottom: 2%;
+  padding-right: 7%;
+}
+
+.main-menu-ul li.disapper {
+  opacity: 0;
+}
```

Comment:
- [ ] One of the issues that caused the horizontal scroll bar to appear was this: kindly don't use `opacity` but use this instead: `display: none`, opacity will only hide the element but it will still be there when you use what I specified it will be gone and not just hidden.